Python/Module5_OddsAndEnds/WorkingWithFiles.md

```diff
@@ -249,6 +249,102 @@ with open("a_poem.txt", mode="r") as my_open_file:
 ```
 <!-- #endregion -->
 
+<!-- #region -->
+## Working with Comma Seperated Value Files
+
+Comma Seperated Value (CSV) files are commonly used to store data that you might typically find in a table. These files can be formatted in many ways, but the typical format is to have each of the column values in the table be separated by commas while having a newline separate each row. Suppose we have the following table of test scores:
+
+|         | Exam 1 (%)           | Exam 2 (%) |
```

Comment:
I'm sure this renders fine but my eyes are bleeding...can you make this look good?